MonoGame.Framework/Graphics/Vertices/VertexBuffer.cs

```diff
@@ -54,13 +54,18 @@ internal protected override void GraphicsDeviceResetting()
 
         public void GetData<T> (int offsetInBytes, T[] data, int startIndex, int elementCount, int vertexStride) where T : struct
         {
+            var elementSizeInBytes = Marshal.SizeOf(typeof(T));
+
+            if (vertexStride == 0)
+                vertexStride = elementSizeInBytes;

```

Comment:
I couldn't think of shorter ways of saying what I wanted to say, without resorting to "specifies the {parameterName}" type comments, which don't help anybody. And when you seem them rendered in a tooltip, without all the markup, I don't think they're too bad. I'm open to suggestions though!